### DISCUSSION TITLE
P1001: Can't reach database server at `aws.connect.psdb.cloud`:`3306

### DISCUSSION THREAD

**Discussion by AbolfazlR5** [planetscale/discussion#520] [Bug Reports]
Hello everyone, I have just started working with Prisma and modified some user information for my website. Now that I'm trying to push it to the database with this command `npx prisma db push` I'm getting this error P1001: Can't reach database server at `aws.connect.psdb.cloud`:`3306` I've also tried the `connect_timeout=30` and still the same error pops up

- This is my code in .env : `DATABASE_URL='mysql://<User>:<Pass>@aws.connect.psdb.cloud/bolimber-saas?connect_timeout=30'`

**Comment by danieloemenike** [comment]
Hi,
I m also experiencing the same issue as you but you shouldn't post your actual db name and password on open forums like this. Maybe try removing that and wait for a solution?

**Comment by Jhuy02** [comment]
I'm also facing a problem like you, I don't know if you've solved it yet. If you have, could you please share it with me?

**Reply by danieloemenike** [reply]
Personally it took just over 1 day for mine to resolve. I tried a lot of solutions which i would share below:
- Deactivate your windows security temporarily, if you use windows. It might be blocking your connection
- Verify that your visual studio code has access to connect with the internet because it is sometimes restricted by windows security.
- Increase the connect timeout in your database url connection to planetscale and try attaching an ssl certificate
- Use powershell or cmd to see if you can connect to other remote databases asides planetscale.
- Update your windows drivers.
- Try Using a different wifi connection to your pc. (I changed my wifi connection and then it suddenly resolved).
- Alternatively try downloading planetscale cli, the name is pscale and use scoop through powershell or use docker image instead
- Lastly be patient and keep trying to see if it works.

**Reply by Jhuy02** [reply]
Thank you
Do you know how to solve this problem?
Error: P1011: Error opening a TLS connection: A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider. (os error -2146762487)

**Reply by danieloemenike** [reply]
This error appears to be a certificate issue for a secure connection to the database. The certificate you are using is not trusted.
Try downloading a certificate and linking it to your database connection in your env file in visual studio code.
PlanetScale already addressed the issue of secure connection here: [https://planetscale.com/docs/concepts/secure-connections](url)
